pyanaconda/modules/payload/live/live_os.py

```diff
@@ -75,6 +86,17 @@ def set_image_path(self, image_path):
         self.image_path_changed.emit()
         log.debug("LiveOS image path is set to '%s'", self._image_path)
 
+    @property
+    def space_required(self):
+        """Get space required for the source image.
+
+        TODO: Add missing check if source is ready. Until then you shouldn't call this when
+        source is not ready.
+
+        :rtype: int
```

Comment:
I miss the information about the units.

Reply:
Fixed.

Reply:
I think that the size is in bytes, because it returns `kilobytes * 1024`.

Reply:
Yeah, you are correct, I missed that before :(.